linode_api4/linode_client.py

```diff
@@ -871,6 +871,213 @@ def users(self, *filters):
         """
         return self.client._get_and_filter(User, *filters)
 
+    def logins(self):
+        """
+        Returns a collection of successful logins for all users on the account during the last 90 days.
+
+        API Documentation: https://www.linode.com/docs/api/account/#user-logins-list-all
+
+        :returns: A list of Logins on this account.
+        :rtype: PaginatedList of Login
+        """
+
+        return self.client._get_and_filter(Login)
+
+    def maintenance(self):
+        """
+        Returns a collection of Maintenance objects for any entity a user has permissions to view. Cancelled Maintenance objects are not returned.
+
+        API Documentation: https://www.linode.com/docs/api/account/#user-logins-list-all
+
+        :returns: A list of Maintenance objects on this account.
+        :rtype: List of Maintenance objects as MappedObjects
+        """
+
+        result = self.client.get(
+            "{}/maintenance".format(Account.api_endpoint), model=self
+        )
+
+        return [MappedObject(**r) for r in result["data"]]
+
+    def payment_methods(self):
+        """
+        Returns a  list of Payment Methods for this Account.
+
+        API Documentation: https://www.linode.com/docs/api/account/#payment-methods-list
+
+        :returns: A list of Payment Methods on this account.
+        :rtype: PaginatedList of PaymentMethod
+        """
+
+        return self.client._get_and_filter(PaymentMethod)
+
+    def add_payment_method(self, data, is_default, type):
+        """
+        Adds a Payment Method to your Account with the option to set it as the default method.
+
+        API Documentation: https://www.linode.com/docs/api/account/#payment-method-add
+
+        :param data: An object representing the credit card information you have on file with
+                     Linode to make Payments against your Account.
+        :type data: dict
+
+        Example usage::
+           data = {
+                "card_number": "4111111111111111",
+                "expiry_month": 11,
+                "expiry_year": 2020,
+                "cvv": "111"
+            }
+
+        :param is_default: Whether this Payment Method is the default method for
+                           automatically processing service charges.
+        :type data: bool
+
+        :param type: The type of Payment Method. Enum: ["credit_card]
+        :type data: str
+        """
+
+        if type != "credit_card":
+            raise ValueError("Unknown Payment Method type: {}".format(type))
+
+        if (
+            "card_number" not in data["data"]
```

Comment:
I think it should be `"card_number" not in data`, right?

Reply:
Yes, you're right. My mistake!